### PR TITLE
Feature/557 bin ts changes

### PR DESCRIPTION
**Pull request by rma-rripken**

### CODE REVIEW

**Review by MikeNeilson — COMMENTED** [review]
Looks reasonable to me.

**Comment by rma-rripken** [comment]
This is related to #557

File: cwms-data-api/src/main/java/cwms/cda/api/enums/VersionType.java [hunk 1]
```diff
@@ -5,20 +5,20 @@
 @Schema(
         name = "Version Type",
         description = "Version type specifies the type of timeseries response to be received. Can be max aggregate "
-                + "or single version. Max aggregate cannot be run if version date field is specified."
+                + "or single version. Max aggregate cannot be ran if version date field is specified."
```
Comment (MikeNeilson) [inline cwms-data-api/src/main/java/cwms/cda/api/enums/VersionType.java]: I think this just got missed or mixed up but I'm pretty sure we decide this *should* be `cannot be run`

File: cwms-data-api/src/test/java/cwms/cda/api/VersionedTimeseriesControllerTestIT.java [hunk 2]
```diff
@@ -29,13 +30,11 @@
...
 public class VersionedTimeseriesControllerTestIT extends DataApiTestIT {
 
-    private static final String OFFICE_STR = "SPK";
-    private static TestAccounts.KeyUser user = TestAccounts.KeyUser.SPK_NORMAL;
+    private static final String OFFICE_STR = "SWT";
     private static final String locationId = "TsVersionedTestLoc";
     private static final String tsId = locationId + ".Flow.Inst.1Hour.0.raw";
     private static final String BEGIN_STR = "2008-05-01T15:00:00Z";
     private static final String END_STR = "2008-05-01T17:00:00Z";
-    public static final String resourcePath = String.format("/cwms/cda/api/%s/", OFFICE_STR.toLowerCase());
```
Comment (MikeNeilson) [inline cwms-data-api/src/test/java/cwms/cda/api/VersionedTimeseriesControllerTestIT.java]: This seems like a good idea, i suspect you may want to pull the latest changes into your fork and rebase.

**Review by MikeNeilson — COMMENTED** [review]
Looking good, I think you're branch/fork is missing some recent changes though.

File: cwms-data-api/src/main/java/cwms/cda/api/enums/VersionType.java (hunk 1 above)
Comment (rma-rripken) [reply cwms-data-api/src/main/java/cwms/cda/api/enums/VersionType.java]: Yes.  Fixed.

File: cwms-data-api/src/test/java/cwms/cda/api/VersionedTimeseriesControllerTestIT.java (hunk 2 above)
Comment (rma-rripken) [reply cwms-data-api/src/test/java/cwms/cda/api/VersionedTimeseriesControllerTestIT.java]: Yes, I needed to rebase on develop.  I wasn't intentionally trying to delete that line.

**Review by MikeNeilson — DISMISSED** [review]

**Review by MikeNeilson — APPROVED** [review]

**Pull request merged**